packages/data-contract/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "data-contract",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js"
```

Comment:
Let's make it "dataContract:deploy", just something different than start, because I feel like we are going to have multiple commands like that (at least for deploying and updating data contract in case schema changes)